.env

```diff
@@ -0,0 +1,2 @@
+TEXTURE_COMMIT=963771408e9ae15c9610726c7d8ac400194921ed
```

Comment:
these may be overridden as environment variables when calling `docker-compose build`